src/supporting-data/citations/adapters/citationsConverter.ts

```diff
@@ -0,0 +1,56 @@
+export type Statistics = {
+  computationallyMappedProteinCount: number;
+  largeScale?: boolean;
+  communityMappedProteinCount: number;
+  reviewedProteinCount: number;
+  unreviewedProteinCount: number;
+};
+
+export enum CitationXRefDB {
+  PubMed = 'PubMed',
+  DOI = 'DOI',
+}
+
+export type CitationXRefDBType = `${CitationXRefDB}`;
+
+export type CitationXRef = {
+  database?: CitationXRefDBType | string;
+  id?: string;
+  // Not sure about all the ones below, copied from somewhere else,
+  // but are they still here?
+  properties?: { [key: string]: string };
+  additionalIds?: string[];
+  isoformId?: string;
+  implicit?: true;
+};
+
+export type Citation = {
+  citationType?: string;
+  authors?: string[];
+  citationCrossReferences?: CitationXRef[];
+  title?: string;
+  publicationDate?: string;
+  journal?: string;
+  firstPage?: string;
+  lastPage?: string;
+  volume?: string;
+  completeAuthorList?: boolean;
+  literatureAbstract?: string;
+  authoringGroup?: string[];
+  submissionDatabase?: string;
+};
+
+export type CitationsAPIModel = {
+  statistics: Statistics;
+  citation: Citation;
```

Comment:
I do believe it would be best if the types regarding the same things were shared somehow. If this is a type related to one of either core or supported data, it should probably be in the corresponding folder, and if anything else it should be in the shared folder.
As in, I believe all the citation types throughout the codebase should come from the type folder of `/supporting-data/citations/` for example.
Thing is, I'm not ready to trust that it's consistent throughout the different endpoints that backend provide, hence the duplication for now.
I believe uniformisation of types is something we should leave for after the alpha release, especially since the endpoints are not stable yet and we don't really have time to spare